ql/src/java/org/apache/hadoop/hive/ql/exec/tez/TezTask.java

```diff
@@ -351,6 +356,17 @@ private String getUserNameForGroups(SessionState ss) {
     return ss.getUserName();
   }
 
+  private static String getHostNameIP() {
+    try {
+      InetAddress ip = InetAddress.getLocalHost();
+      String hostname = ip.getHostName();
+      return String.format("%s (%s)", hostname, ip.getHostAddress());
+    } catch (UnknownHostException e) {
+      LOG.debug("Unable to determine hostname", e);
```

Comment:
This should be at least info, or rather warn level.

Reply:
@miklosgergely The issue here is that anything that logs to this logger gets forwarded to the user's connection.  I don't want to print errors and stack traces to their connection.  If user starts getting "unknown" responses, they can enable debug logging and get it.